test/client/Mutation.test.tsx

```diff
@@ -1500,7 +1502,7 @@ it('does not update state after receiving data after it has been unmounted', don
         return null;
       } else {
         return (
-          <Mutation mutation={mutation}>
+          <Mutation mutation={mutation} onCompleted={onCompleted}>
```

Comment:
I used an existing test just to get something out there and to make sure it's working as expected. I can also add a separate test for this use case. Please let me know.

Reply:
Would you mind extracting this to a seperate test? Improves error reporting